README.md

```diff
@@ -8,63 +8,73 @@
 
 ## Table of Contents
 
-- [Requirements](#requirements)
-- [Installation](#installation)
-  - [Platform Packages](#platform-packages)
+- [Installing](#installing)
+  - [Requirements](#requirements)
+  - [From sources](#from-sources)
   - [Godep](#godep)
+  - [Platform Packages](#platform-packages)
   - [Cross Compilation](#cross-compilation)
-- [Configuration](#configuration)
-- [Usage](#usage)
+  - [API keys](#api-keys)
+  - [Scripts](#scripts)
+- [Using](#using)
+  - [Hyphens](#hypens)
   - [Initializing](#initializing)
   - [De Initializing](#de-initializing)
-  - [End to End Encryption](#end-to-end-encryption)
   - [Traversal Depth](#traversal-depth)
+  - [Configuring General Settings](#configuring-general-settings)
+  - [Excluding And Including Objects](#excluding-and-including-objects)
+    - [Sample .driveignore with the exclude and include clauses combined](sample-.driveignore-with-the-exclude-and-include-clauses-combined)
   - [Pulling](#pulling)
+    - [Verifying Checksum] (#verifying-checksum)
     - [Exporting Docs](#exporting-docs)
   - [Pushing](#pushing)
+  - [Pulling And Pushing Notes](#pulling-and-pushing-notes)
+  - [Pulling And Pushing End to End Encryption](#pulling-and-pushing-end-to-end-encryption)
```

Comment:
Let's keep this as `End to End Encryption`. Sure pulling and pushing is its context and the commands that use it but
`End to End Encryption` without any prefixes advertises the feature
more than `Pulling and Pushing End to End Encryption` which
makes it a little hard to pick up on.